ldes-server-port-admin-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/OpenApiEventStreamsController.java

```diff
@@ -0,0 +1,264 @@
+					<http://localhost:8080/mobility-hindrances> <http://example.org/memberType> <https://data.vlaanderen.be/ns/mobiliteit#Mobiliteitshinder> .
+					<http://localhost:8080/mobility-hindrances> <http://example.org/hasDefaultView> "false"^^<http://www.w3.org/2001/XMLSchema#boolean> .
+					<http://localhost:8080/mobility-hindrances> <https://w3id.org/tree#shape> <http://localhost:8080/mobility-hindrances/shape> .
+					<http://localhost:8080/parcels> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://w3id.org/ldes#EventStream> .
+					<http://localhost:8080/parcels> <https://w3id.org/ldes#timestampPath> <http://purl.org/dc/terms/created> .
+					<http://localhost:8080/parcels> <https://w3id.org/ldes#versionOfPath> <http://purl.org/dc/terms/isVersionOf> .
+					<http://localhost:8080/parcels> <http://example.org/memberType> <https://basisregisters.vlaanderen.be/implementatiemodel/gebouwenregister#Perceel> .
+					<http://localhost:8080/parcels> <http://example.org/hasDefaultView> "true"^^<http://www.w3.org/2001/XMLSchema#boolean> .
+					<http://localhost:8080/parcels> <https://w3id.org/tree#shape> <http://localhost:8080/shape> .
+					<https://w3id.org/ldes#view> <http://localhost:8080/pagination> ""^^<http://www.w3.org/2001/XMLSchema#integer> .
+					_:genid2 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://example.org/Fragmentation> .
+					_:genid2 <http://example.org/name> "pagination" .
+					_:genid2 <http://example.org/memberLimit> "100" .
+					_:genid1 <http://example.org/fragmentationStrategy> _:genid2 .
+					<http://localhost:8080/parcels/pagination> <https://w3id.org/tree#viewDescription> _:genid1 .
+					<http://localhost:8080/parcels/shape> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/shacl#NodeShape> .
+					<http://localhost:8080/parcels/shape> <http://www.w3.org/ns/shacl#nodeShape> _:genid3 .
+					<http://localhost:8080/parcels/shape> <http://www.w3.org/ns/shacl#deactivated> "true"^^<http://www.w3.org/2001/XMLSchema#boolean> .
+					_:genid3 <http://www.w3.org/ns/shacl#closed> "true"^^<http://www.w3.org/2001/XMLSchema#boolean> .
+					_:genid3 <http://www.w3.org/ns/shacl#propertyShape> _:genid4 .
+					<http://localhost:8080/mobility-hindrances/shape> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/shacl#NodeShape> .
+					<http://localhost:8080/mobility-hindrances/shape> <http://www.w3.org/ns/shacl#nodeShape> _:genid5 .
+					<http://localhost:8080/mobility-hindrances/shape> <http://www.w3.org/ns/shacl#deactivated> "true"^^<http://www.w3.org/2001/XMLSchema#boolean> .
+					_:genid5 <http://www.w3.org/ns/shacl#closed> "true"^^<http://www.w3.org/2001/XMLSchema#boolean> .
+					_:genid5 <http://www.w3.org/ns/shacl#propertyShape> _:genid6 .
+					""")),
+			@Content(mediaType = contentTypeJSONLD, schema = @Schema(implementation = String.class), examples = @ExampleObject(value = """
```

Comment:
The json will be properly formatted in the swagger UI if you remove this part:
`schema = @Schema(implementation = String.class),`